src/config.c

```diff
@@ -229,6 +327,23 @@ CONFIG_GETTER(getMinOperationWorkers) {
   return sdscatprintf(ss, "%lu", config->minOperationWorkers);
 }
 
+// min-operation-workers
+int set_min_operation_workers(const char *name,
```

Comment:
can't you use one of the generic type setters at the start of this file?
Is this custom function needed?
Doesn't seem like it does any validation.

Reply:
Needs to also call `workersThreadPool_SetNumWorkers`